File: sample/javaScript/mainMakePlyModelSample.mjs
import { Voxelamming, getBoxesFromPly } from 'voxelamming';
// import { getBoxesFromPly } from 'voxelamming'; // test
// import Voxelamming from './voxelamming.js';  // test

const roomName = '1000';
const vox = new Voxelamming(roomName);

vox.setBoxSize(0.5);
vox.setBuildInterval(0.01);

const plyFileName = '../ply_file/piyo.ply'

const boxes = getBoxesFromPly(plyFileName)

for (const box of boxes) {
  // console.log(b)
  vox.createBox(...box)
}

await vox.sendData();
console.log('send data done');

File: sample/python/main_make_ply_model_sample.py
# voxelammingパッケージからVoxelammingクラスをインポートします
from voxelamming import Voxelamming, get_boxes_from_ply
# from voxelamming_local import Voxelamming, get_boxes_from_ply

ply_file_name = '../ply_file/piyo.ply'

# Voxelammingアプリに表示されている部屋名を指定してください
room_name = "1000"
# Voxelammingクラスのインスタンスを生成します
vox = Voxelamming(room_name)

# ボクセルの設定を行います
vox.set_box_size(1)
vox.set_build_interval(0.01)

# ボクセルを配置するため、位置と色を設定します
boxes = get_boxes_from_ply(ply_file_name)

for box in boxes:
    vox.create_box(*box)

# ボクセルデータをアプリに送信します。
vox.send_data("main_make_model_sample")
